nvhpc/tasks/install.yaml

```diff
@@ -0,0 +1,37 @@
+#- name: Clear Previous NVHPC SDK v{{ NSP_nvhpc_version }} Toolkit Run Script
+#  ansible.builtin.file:
+#    path: "{{ [NSP_scratch_directory, 'nvhpc-{}'.format(NSP_nvhpc_version)] | path_join }}"
+#    state: absent
+#  when: NSP_nvhpc_clean_build and [NSP_scratch_directory, 'nvhpc-{}'.format(NSP_nvhpc_version)] | path_join is directory
+#  tags:
+#    - cuda
+#    - installation
+#
+- name: "Download NVHPC SDK v{{ NSP_nvhpc_version }} Toolkit Run Script"
+  ansible.builtin.get_url:
+    url: "{{ NSP_nvhpc_url }}"
+    dest: "{{ NSP_scratch_directory }}"
+    owner: "{{ NSP_user }}"
+    group: "{{ NSP_group }}"
+    mode: "744"
+  tags:
+    - cuda
+    - installation
+
+- name: "Make sure destination directory exists"
+  ansible.builtin.file:
+    path: "{{ [NSP_install_root, 'nvhpc', NSP_nvhpc_version] | path_join }}"
+    state: directory
+
+- name: "Unarchive tar file"
+  ansible.builtin.unarchive:
+    src: "{{ [NSP_scratch_directory, NSP_nvhpc_file ~.tar.gz] | path_join }}"
+    dest: "{{ NSP_scratch_directory}}"
+    remote_src: yes
+
+- name: "Install"
+  ansible.builtin.command:
+    cmd: |
+      cp "{{ [NSP_scratch_directory, NSP_nvhpc_file, 'install_components/Linux_x86_64/', NSP_nvhpc_version] | path_join }}" "{{ [NSP_install_root, 'nvhpc', NSP_nvhpc_version] | path_join }}/."
```

Comment:
Is `cp` the same as the steps defined by NVIDIA's docs -- https://docs.nvidia.com/hpc-sdk/hpc-sdk-install-guide/index.html#installation-steps-for-linux ?